cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -222,6 +228,11 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
             }
         }
 
+        packageIds?.also {
```

Comment:
I would consider `if(packageIds != null)` more straightforward to read.

Reply:
That's what I had at first, but then got a hint that the smart-cast on a Clikt option (which is a delegated property) would not work, so I went for this syntax.